tools/db_stress.cc

```diff
@@ -1316,6 +1334,21 @@ class StressTest {
   }
 
  private:
+  void ASSERT_SAME(DB* db, ColumnFamilyHandle* cf,
+                   ThreadState::SnapshotState& snap_state) {
+    ReadOptions ropt;
+    ropt.snapshot = snap_state.snapshot;
+    PinnableSlice exp_v(&snap_state.value);
+    exp_v.PinSelf();
+    Status s;
+    PinnableSlice v;
+    s = db->Get(ropt, cf, snap_state.key, &v);
+    assert(snap_state.status == s);
+    assert(s.ok() || s.IsNotFound());
+    if (s.ok()) {
+      assert(exp_v == v);
```

Comment:
were we planning to check equality using some non-assert mechanism since it'll be disabled by `DEBUG_LEVEL=0`?